### ISSUE TITLE
Support GHC 8.8.3

### ISSUE THREAD

**Issue by asr** [actions/setup-haskell#8]
GHC 8.8.3 was [announced](https://mail.haskell.org/pipermail/ghc-devs/2020-February/018643.html). Using this action I'm getting the error

```
Run actions/setup-haskell@v1
  with:
    ghc-version: 8.8.3
    cabal-version: 3.0
##[error]Version 8.8.3 of ghc not found
```

Blocking https://github.com/agda/agda/issues/4476.

**Comment by asr** [comment]
I created PR https://github.com/actions/virtual-environments/pull/483 (based on PR https://github.com/actions/virtual-environments/pull/303) for fixing this issue.

**Comment by fosskers** [comment]
Any updates here?

**Comment by asr** [comment]
The PR https://github.com/actions/virtual-environments/pull/483 was merged. It seems we need to wait for a release of [Ubuntu 16.04](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1604-README.md) and/or [Ubuntu 18.04](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md) from [virtual environments](https://github.com/actions/virtual-environments).

**Comment by fosskers** [comment]
It seems that I'm able to set `ghc-version: 8.8.2` with this repo's `v1`, but it can't seem to find `8.8.3` yet.

**Comment by asr** [comment]
Same here.

**Comment by fosskers** [comment]
Oh, it looks like both `8.8.3` and `stack` are included in the 18.04 prerelease that was released 4 days ago. Not sure if that's what corresponds to `ubuntu-latest` in our CI config, though. Probably not yet.

**Comment by asr** [comment]
From [virtual environments](https://github.com/actions/virtual-environments) currently `ubuntu-latest` is Ubuntu 18.04.

I just could use GHC 8.8.3 on Ubuntu 16.04 (which is green in the Latest Release & Rollout Progress column).

**Comment by asr** [comment]
GHC 8.8.3 was included in both Ubuntu 16.04 and 18.04.
